backend/env.yml

```diff
@@ -16,11 +16,12 @@ dev:
   LOGIN_GOV_ISSUER: ${env:LOGIN_GOV_ISSUER}
   FARGATE_SG_ID: ${env:FARGATE_SG_ID}
   FARGATE_SUBNET_ID: ${env:FARGATE_SUBNET_ID}
-  FARGATE_MAX_CONCURRENCY: 5
+  FARGATE_MAX_CONCURRENCY: '${env:FARGATE_MAX_CONCURRENCY}'
```

Comment:
@cablej do we even need the dev env variables in serverless.yml? It appears we're no longer using it, but let me know if you think we need it.